acts-as-taggable-on.gemspec

```diff
@@ -15,20 +15,18 @@ Gem::Specification.new do |gem|
   gem.files         = `git ls-files`.split($/)
   gem.test_files    = gem.files.grep(%r{^spec/})
   gem.require_paths = ['lib']
-  gem.required_ruby_version     = '>= 3.0.0'
+  gem.required_ruby_version     = '>= 3.2.0'
 
   if File.exist?('UPGRADING.md')
     gem.post_install_message = File.read('UPGRADING.md')
   end
 
-  gem.add_runtime_dependency 'activerecord', '>= 7.0', '< 8.0'
+  gem.add_runtime_dependency 'activerecord', '>= 7.1', '< 8.1'
   gem.add_runtime_dependency 'zeitwerk', '>= 2.4', '< 3.0'
 
   gem.add_development_dependency 'rspec-rails'
   gem.add_development_dependency 'rspec-its'
   gem.add_development_dependency 'rspec'
   gem.add_development_dependency 'barrier'
   gem.add_development_dependency 'database_cleaner'
-
-  gem.metadata['changelog_uri'] = gem.homepage + '/blob/master/CHANGELOG.md'
```

Comment:
@matthewbjones 
Was this line removed intentionally? It was added in [PR#1133](https://github.com/mbleigh/acts-as-taggable-on/pull/1133) to provide a Changelog link in rubygems.org.

Reply:
> Let readd it.

Sorry, I don't understand?